Makefile

```diff
@@ -4,8 +4,19 @@ DEPS_PKGNAME=leapp-el7toel8-deps
 VERSION=`grep -m1 "^Version:" packaging/$(PKGNAME).spec | grep -om1 "[0-9].[0-9.]**"`
 DEPS_VERSION=`grep -m1 "^Version:" packaging/$(DEPS_PKGNAME).spec | grep -om1 "[0-9].[0-9.]**"`
 ACTOR_PATH=repos
+LIBRARY_PATH=`python utils/library_path.py $(ACTOR)`
 ifdef ACTOR
 	ACTOR_PATH=`python utils/actor_path.py $(ACTOR)`
+	TEST_ACTOR_ARG=--actor=$(ACTOR)
+	ifndef TEST_LIBS
+		LIBRARY_PATH=
+	endif
+endif
+ifdef TEST_LIBS
```

Comment:
I think it would be better if we had an expected value for TEST_LIBS because now you can write `TEST_LIBS=false make test` and it will only test libraries, which is confusing